src/mlpack/methods/naive_bayes/nbc_main.cpp

```diff
@@ -112,8 +111,11 @@ PARAM_FLAG("incremental_variance", "The variance of each class will be "
 
 // Test parameters.
 PARAM_MATRIX_IN("test", "A matrix containing the test set.", "T");
+// parameter output is to be deprecated and removed in mlpack 4.0
 PARAM_UROW_OUT("output", "The matrix in which the predicted labels for the"
     " test set will be written.", "o");
+PARAM_UROW_OUT("predictions", "The matrix in which the predicted labels for the"
+    " test set will be written.", "a");
 PARAM_MATRIX_OUT("output_probs", "The matrix in which the predicted probability"
     " of labels for the test set will be written.", "p");
```

Comment:
We should also deprecate `output_probs` and provide a `probabilities` output parameter.